keyboards/czmao/ds17r2/keyboard.json

```diff
@@ -0,0 +1,141 @@
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "dynamic_keymap": {
+      "layer_count": 8
+    },
+    
+
+    
+    
+    "matrix_pins": {
+      "cols": ["B12", "B14", "B15", "A8"],
+      "rows": ["A9", "A10", "A15",  "B3",  "B4"]
+    },
+
+    "ws2812": {
+      "pin": "B13"
+    },
+
+    "rgb_matrix": {
+        "animations": {
+            "gradient_left_right": true,
+            "breathing": true,
+            "band_sat": true,
+            "band_val": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_up_down": true,
+            "rainbow_moving_chevron": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "typing_heatmap": true,
+            "solid_reactive_simple": true,
+            "solid_reactive": true,
+            "solid_reactive_wide": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_multinexus": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_multisplash": true
+        },
+        "driver": "ws2812",
+        "max_brightness":180,
+        "layout": [
+        { "flags": 2, "matrix": [0, 0], "x": 0, "y": 0 },
+        { "flags": 2, "matrix": [0, 1], "x": 75, "y": 0 },
+        { "flags": 2, "matrix": [0, 2], "x": 150, "y": 0 },
+        { "flags": 2, "matrix": [0, 3], "x": 224, "y": 0 },
+
+        { "flags": 2, "matrix": [1, 3], "x": 224, "y": 32 },
+        { "flags": 2, "matrix": [3, 3], "x": 224, "y": 50 },
+        { "flags": 2, "matrix": [4, 2], "x": 150, "y": 64 },
+        { "flags": 2, "matrix": [4, 0], "x": 100, "y": 64 },
+
+        { "flags": 2, "matrix": [3, 0], "x": 0, "y": 48 },
+        { "flags": 2, "matrix": [2, 0], "x": 0, "y": 32 },
+        { "flags": 2, "matrix": [1, 0], "x": 0, "y": 16 },
+        { "flags": 2, "matrix": [1, 1], "x": 75, "y": 16 },
+
+        { "flags": 2, "matrix": [1, 2], "x": 150, "y": 16 },
+        { "flags": 2, "matrix": [2, 2], "x": 150, "y": 32 },
+        { "flags": 2, "matrix": [3, 2], "x": 150, "y": 48 },
+        { "flags": 2, "matrix": [3, 1], "x": 75, "y": 48 },
+
+        { "flags": 2, "matrix": [2, 1], "x": 75, "y": 32 }
+        ],
+        "sleep": true
+    },
+
+    "layouts": {
+      "LAYOUT_numberpad_5x4": {
```

Comment:
Would the [default 5x4 numpad layout](https://github.com/qmk/qmk_firmware/tree/master/layouts/default/numpad_5x4) work here?

Reply:
sorry not the same layout